mllib/src/main/scala/org/apache/spark/ml/feature/QuantileDiscretizer.scala

```diff
@@ -67,22 +68,6 @@ private[feature] trait QuantileDiscretizerBase extends Params
   /** @group getParam */
   def getNumBucketsArray: Array[Int] = $(numBucketsArray)
 
-  /**
-   * Relative error (see documentation for
-   * `org.apache.spark.sql.DataFrameStatFunctions.approxQuantile` for description)
-   * Must be in the range [0, 1].
-   * Note that in multiple columns case, relative error is applied to all columns.
```

Comment:
Nit: Seems the above line got removed in the new documentation. I guess maybe put it somewhere else in the doc? Maybe put it  in the end of line 97?

``` Since 2.3.0, `QuantileDiscretizer` can map multiple columns at once by setting the `inputCols`
 parameter. If both of the `inputCol` and `inputCols` parameters are set, an Exception will be
 thrown. To specify the number of buckets for each column, the `numBucketsArray` parameter can
be set, or if the number of buckets should be the same across columns, `numBuckets` can be
set as a convenience. Note that in multiple columns case, relative error is applied to all columns.```